app/js/displayCategories.ts

```diff
@@ -0,0 +1,18 @@
+function populateCategories() {
+    document.querySelector('#categoriesContainer').innerHTML = '';
+    populateHandlebars('#categoriesContainer', 'js/templates/categoryItem.hbs', 'category')
+        .then(() => {
+            document.querySelectorAll('.deleteCategory').forEach((button: HTMLElement) => {
+                button.addEventListener('click', async(clickedBtn: any) => {
```

Comment:
You've type hinted the parameter in this async anonymous function to `any`... Does it REALLY accept a value with any kind of data type? Can I pass a function here? 

Think about the value that is actually being passed here and type accordingly :). It's tempting to use `any` as it makes your TS compile so you can move on with your life rather than think about what your code is actually doing, but when you go on to use TS professionally - this kind of imprecision would not be allowed.